tools/toolchain.cmake.export

```diff
@@ -23,11 +23,16 @@ file(GLOB STARTUP_OBJS ${NUTTX_PATH}/startup/*)
 add_compile_options(-nostdlib)
 add_compile_options(-ffunction-sections -fdata-sections)
 
+set(ENTRY_NAME "__start")
+if("${NUTTX_MODE}" STREQUAL "k")
+  set(ENTRY_NAME "_start")
```

Comment:
could we modify the code  to make the entry point consistent?

Reply:
That will propogate to multiple startup and linker sript files, which is more changes than this patch.